File: w01/d04/Carlos_Pichardo/calculator/calculator.rb
class Calculator

	def add(x,y)
		x + y
	end 

	def subtract(x,y)
		x - y
	end

	def divide(x,y)
		if y == 0
			puts "NaN"
		else
			x.to_f / y.to_f
		end
	end

	def multiply(x,y)
		x * y
	end

end 

File: w07/d05/Ben_Karl/jasmine_drill/src/calculator.js
function calculator(num1, num2){
  this.num1 = num1;
  this.num2 = num2;
};

calculator.prototype.add = function(){
  return this.num1 + this.num2;
};

// calculator.prototype.subtract = function(){
//   return this.num1 - this.num2;
// };

// calculator.prototype.multiply = function(){
//   return this.num1 * this.num2;
// }
